configure-nonroot.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/bash
+
+set -eux
+
+NONROOT_USER="nonroot"
+NONROOT_GROUP="nonroot"
```

Comment:
I'm quite sure the RPMs already set up an "ironic" user and group, why not use them? "nonroot" looks confusing to me.

Reply:
`nonroot` is standard user in k8s rootless. I can change to `ironic`, no problem.